app/app/Http/Controllers/SocialAuthController.php

```diff
@@ -25,11 +25,12 @@ public function getSocialLogin($providerName) {
 			$profile = $provider->getUserProfile();
 			$provider->logout();
 			$this->createNewUserWithGoogle($profile);
-			return view('pages.signup.success', ['email' => $profile->email]);
+			session(['email' => $profile->email]);
+			return redirect()->intended('profile');
 
 		} catch (Exception $e) {
-			// 	echo "Ooophs, we got an error: " . $e->getMessage();
-			//	echo " Error code: " . $e->getCode();
+				// echo "Ooophs, we got an error: " . $e->getMessage();
+				// echo " Error code: " . $e->getCode();
 			return view('pages.signin')->withErrors('Ooophs, there is a problem! you can try it again.')->with('email','');
```

Comment:
Remove the commented echo statements since they look unhelpful to anyone reading the code.